WordPress/Classes/Extensions/Dictionary+JSONPrettify.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+extension Dictionary where Key == String, Value == AnyObject {
+    func prettyPrint() -> String {
```

Comment:
This is probably a cool thing to have, but I didn't see it being used anywhere.
I'm just highlighting it in case you forgot to remove it. 🙂